config/locales/en.yml

```diff
@@ -257,6 +257,9 @@ en:
               reused:
                 one: "has been used before. Please choose one that is different from your last one."
                 other: "has been used before. Please choose one that is different from your last %{count}."
+              match: |
+                Confirm new password. <br>
```

Comment:
My problem is not with the fact that it works - my problem is the fact that labels and layout are separate concepts. You do not (I repeat, you do __not__) style text flow via html tags within the text (like using empty `<p>&nbsp;</p>` to achieve more whitespace.

Also, this will make it more obscure to identify the source of the tag.